src/theme/Footer/styles.module.css

```diff
@@ -1,41 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
-.footerLogoLink {
-  opacity: 0.5;
-  transition: opacity 0.15s ease-in-out;
-}
-
-.footerLogoLink:hover {
-  opacity: 1;
-}
```

Comment:
These first two styles are from the classic theme, we can delete them because the theme takes care of them for us.